cluster/images/hyperkube/kube-proxy.json

```diff
@@ -13,7 +13,8 @@
                 "proxy",
                 "--master=http://127.0.0.1:8080",
                 "--v=2",
-                "--resource-container=\"\""
+                "--resource-container=\"\"",
+                "--proxy-mode=iptables"
```

Comment:
I think we should keep configuration as consistent as possible. Please remove this change and update PR description.